README.md

```diff
@@ -1,2 +1,269 @@
-# cognitive-social-crm
-An application that monitors a Twitter feed and determines customer sentiment using IBM Watson Assistant, Tone Analyzer, Natural Language Understanding, as well as CloudantDB
+*Read this in other languages: [日本語](README-ja.md).*
+
+[![Build Status](https://travis-ci.org/IBM/cognitive-social-crm.svg?branch=master)](https://travis-ci.org/IBM/cognitive-social-crm)
+
+# Monitor Twitter feeds to better understand customer sentiment using Watson Assistant, Tone Analyzer, and Natural Language Understanding
+
+> Watson Conversation is now Watson Assistant. Although some images in this code pattern may show the service as Watson Conversation, the steps and processes will still work.
```

Comment:
I think this can now be removed. I don't think you are using any old images with Watson Conversation.